docker/apollo-server/type.ts

```
@@ -1,4 +1,4 @@
-import { GraphQLResolveInfo } from 'graphql';
+﻿import { GraphQLResolveInfo } from 'graphql';
```

ESLint

[error] Unexpected Unicode BOM (Byte Order Mark).

(unicode-bom)